packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -36,7 +36,7 @@
 		"test:end-to-end": "npm run test:jest",
 		"test:jest": "jest",
 		"test:jest:verbose": "cross-env FLUID_TEST_VERBOSE=1 jest",
-		"test:mocha:fail": "npm run mocha",
+		"test:mocha": "npm run mocha",
```

Comment:
What was the intention of initially naming the script `:fail`?

Reply:
Per Curtis' [PR that made the change](https://github.com/microsoft/FluidFramework/pull/17082):

> Also, disable test:mocha script in devtools-view and devtools-browser-extension as they are broken (and wasn't ran in the CI previously because they don't have test:mocha:multireport script.